references.md

```diff
@@ -2,7 +2,7 @@
 
 This chapter contains links to reference manuals and sheets, including:
 
-- [Deno Standard APIs](https://deno.land/api@v$STD_VERSION)
+- [Deno Standard APIs](https://deno.land/api@$CLI_VERSION)
```

Comment:
```suggestion
- [Deno Standard APIs](https://deno.land/api@v$CLI_VERSION)
```

Reply:
That actually broke the link again (since `$CLI_VERSION` is already prefixed with a `v`)

Reply:
Oops! Sorry I didn't realize it

Reply:
There were many same mistakes https://github.com/denoland/manual/pull/439